core/events.js

```diff
@@ -866,6 +868,146 @@ Blockly.Events.Move.prototype.run = function(forward) {
   }
 };
 
+/**
```

Comment:
FYI I'm eventually going to split this file up: https://github.com/google/blockly/issues/1618

No change needed in this PR though.